### ISSUE TITLE
Is that possible to only use the filter? 

### ISSUE THREAD

**Issue by ItsRay** [aiyaapp/AiyaEffectsAndroid#71]
I want to ask that is that possible to only use the filter?
Because I want to open camera by myself instead of by your service.
Is this possible? And is there a document for this usage?

**Comment by aiyaapp** [comment]
Yes. You can modify the java code of sdk, instead of pull the libs directly.
